mlir/include/mlir/Dialect/Vector/Transforms/VectorDistribution.h

```diff
@@ -75,10 +75,18 @@ using WarpShuffleFromIdxFn =
 /// Collect patterns to propagate warp distribution. `distributionMapFn` is used
 /// to decide how a value should be distributed when this cannot be inferred
 /// from its uses.
+///
+/// Added control over the pattern benefit for propagating
```

Comment:
The separate control over `vector.transfer_read` op pattern benefit is to ensure ..